### ISSUE TITLE
[Need a fix] Migrations exit with code 1 when no migrations are left to run

### ISSUE THREAD

**Issue by dashsatish** [piccolo-orm/piccolo#28]
Hi,
Currently `piccolo migrations` exits with code `1` even if there are no migrations left to run.

Output:
```
(my-service) ➜ pipenv run piccolo migrations forwards my_service all
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
Loading .env environment variables…
Running Postgres version 11.1
Running migrations ...
All migration ids = ['2020-12-15T18:55:12', '2020-12-16T19:54:32', '2020-12-17T10:58:36', '2020-12-17T16:01:04', '2020-12-17T16:15:15', '2020-12-17T16:19:27', '2020-12-17T20:23:06']
Haven't run = []
No migrations left to run!
(my-service) ➜  echo $?
1
```

Can this be made defensive so that the cli returns a code `0` instead of exiting with `1`?? Exit 1 kind of indicates one or migrations have failed.

**Comment by dantownsend** [comment]
@dashsatish That's a good point - I've created a fix, and will do a release later today.

https://github.com/piccolo-orm/piccolo/pull/29

**Comment by dantownsend** [comment]
@dashsatish Fixed in version 0.14.9, now on PyPI. Thanks for reporting this issue!
